### PR TITLE
add schedule sample of CLI/SDK

### PR DESCRIPTION
**Pull request by eniac871**
# PR into Azure/azureml-examples

## Checklist

add schedule sample of CLI/SDK. Currently the sample is for schedule a pipeline job, and the related schedule CRUD operations

- [ ] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: .github/workflows/cli-schedules-cron-schedule.yml
Comment (wangchao1230) [inline .github/workflows/cli-schedules-cron-schedule.yml:25]: I think we manually created these workflows for schedule right?

Make sure this won't be overrided by the auto-gen script
cli/readme.py 
sdk/readme.py

File: cli/schedules/README.md
Comment (wangchao1230) [inline cli/schedules/README.md:2]: cli/README.md

sdk/README.md 

may need to be updated

File: cli/setup.sh
Comment (wangchao1230) [inline cli/setup.sh:9]: mark, need to wait for stable version release before merge this PR

File: sdk/schedules/schedule.ipynb
Comment (wangchao1230) [inline sdk/schedules/schedule.ipynb:230]: we should put all imports in first code scetion

File: sdk/schedules/schedule.ipynb
Comment (wangchao1230) [inline sdk/schedules/schedule.ipynb:301]: May do something like below to have better output 

schedules = ml_client.schedules.list()
[s.name for s in schedules]

File: sdk/schedules/schedule.ipynb
Comment (wangchao1230) [inline sdk/schedules/schedule.ipynb:335]: begin_delete

Comment (eniac871) [reply sdk/schedules/schedule.ipynb]: updated

File: sdk/schedules/schedule.ipynb
Comment (eniac871) [reply sdk/schedules/schedule.ipynb]: make sense, updated

File: sdk/schedules/schedule.ipynb
Comment (eniac871) [reply sdk/schedules/schedule.ipynb]: updated

File: .github/workflows/cli-schedules-cron-schedule.yml
Comment (eniac871) [reply .github/workflows/cli-schedules-cron-schedule.yml]: update: I added the content & workflow generate for cli/readme.py given there're 3 cli samples now. For SDK, I just manual update the readme.md

Comment (eniac871) [reply .github/workflows/cli-schedules-cron-schedule.yml]: And because schedule are new concept, the autogen script won't touch that part if we don't add new autogen code for schedule

File: sdk/schedules/schedule.ipynb
Comment (D-W-) [inline sdk/schedules/schedule.ipynb]: This seems not used

File: sdk/schedules/schedule.ipynb
Comment (D-W-) [inline sdk/schedules/schedule.ipynb:318]: This cell seems empty

File: sdk/schedules/schedule.ipynb
Comment (D-W-) [inline sdk/schedules/schedule.ipynb:1]: Nit: may rename this sample to "recurrence-schedule"?

File: sdk/schedules/schedule.ipynb
Comment (brynn-code) [reply sdk/schedules/schedule.ipynb]: Removed

File: sdk/schedules/schedule.ipynb
Comment (brynn-code) [reply sdk/schedules/schedule.ipynb]: Added

File: sdk/schedules/schedule.ipynb
Comment (brynn-code) [reply sdk/schedules/schedule.ipynb:1]: I think Han means "schedule-with-pipeline-job", in case we will add 'schedule-with-endpoint', etc.

Comment (eniac871) [reply sdk/schedules/schedule.ipynb]: renamed

**Review by D-W- — APPROVED** [review]

**Review by brynn-code — APPROVED** [review]

**Comment by chamalHealthscope** [comment]
Thanks for the feature. When we try it our selves we get the following error.  The error is quite opposing to the use case documentation. Any feedback as to why this might be ? 

(UserError) Job info with trigger is invalid, details: You may only specify a schedule when the Frequency is Day or Week
Code: UserError
Message: Job info with trigger is invalid, details: You may only specify a schedule when the Frequency is Day or Week

**Comment by chamalHealthscope** [comment]
btw, this is the sample code we use in the pipeline scheduler YAMl file, 

####
$schema: https://azuremlschemas.azureedge.net/latest/schedule.schema.json

name: simple_recurrence_job_schedule
display_name: Simple recurrence job schedule
description: a simple hourly recurrence job schedule

trigger:
  type: recurrence
  frequency: Minute  #can be minute, hour, day, week, month
  interval: 15 
create_job: ./pipeline-test.yaml

########

**Comment by brynn-code** [comment]
@chamalHealthscope  Ack, seems a bug, you may use cron expression with expression '*/15 * * * *' to mitigate the issue, thanks for your feed back.

**Comment by brynn-code** [comment]
@chamalHealthscope We have published the new schema, you could take a look via the link, thanks.

**Comment by chamalHealthscope** [comment]
Thanks. Btw we managed to overcome the issue I was coming up with az upgarde.

**Pull request merged**
